lib/util.js

```diff
@@ -27,16 +27,13 @@ var simdFormatters;
 if (typeof global.SIMD === 'object' && global.SIMD !== null) {
   simdFormatters = new Map();
 
-  const make = (extractLane, count) => {
-    return (ctx, value, recurseTimes, visibleKeys, keys) => {
+  const make =
+    (extractLane, count) => (ctx, value, recurseTimes, visibleKeys, keys) => {
       const output = new Array(count);
       for (var i = 0; i < count; i += 1)
```

Comment:
I think we're avoiding using `let` in loops at the moment due to performance concerns. For more context, see https://github.com/nodejs/node/issues/8637 and https://github.com/nodejs/node/pull/9049.